core/src/main/java/org/jruby/ir/interpreter/FullInterpreterContext.java

```diff
@@ -188,6 +189,6 @@ public int determineRPC(int ipc) {
             if (ipc <= rescueIPCs[i]) return rescueIPCs[i + 1];
         }
 
-        throw new RuntimeException("BUG: no RPC found for " + getFileName() + ":" + getName() + ":" + ipc + getInstructions());
+        throw new RuntimeException("BUG: no RPC found for " + getFileName() + ":" + getName() + ":" + ipc + Arrays.toString(getInstructions()));
```

Comment:
This is wrong both ways. The array will just be a mangled `Instr]@0x1234` sort of toString, and the Arrays version will include ALL of the instructions in the error message.

Reply:
Hrm. Perhaps do you want me to replace `getInstructions()` with `rescueIPCs` and Arrays.toString that?

Reply:
@nglorioso just remove getInstructions() altogether.  Originally we only saw this while developing our new runtime so we probably never cared about anything other than ipc since we could just run the program again and look at the instructions on a second run.  If we corrected this to print out all instrs I think users would complain about a gazillion lines of output.

Reply:
As an added note we should consider possibly changing this to be an hserr-like output in addition to notifying the user we hit an internal bug.  We could then dump CFG, instrs, ipc, etc into a file so they could paste that into an issue (sorry for adding this to an incidental PR review but sometimes inspiration strikes!).  I am not asking that here though :)